db/seeds.rb

```diff
@@ -3,8 +3,8 @@
 # TODO: Remove network and school seeding when we have a way of getting them from GIAS
 unless Network.first
   network = Network.create!(name: "Test school network")
-  School.create!(urn: "TEST_URN_1", name: "Test school one", address_line1: "Test address", country: "England", postcode: "TEST1", network: network, domains: %w[testschool1.sch.uk network.com])
-  School.create!(urn: "TEST_URN_2", name: "Test school two", address_line1: "Test address London", country: "England", postcode: "TEST2", network: network, domains: %w[testschool2.sch.uk network.com])
+  School.create!(urn: "TEST_URN_1", name: "Test school one", address_line1: "Test address", country: "England", postcode: "TEST1", network: network, domains: %w[testschool1.sch.uk network.com digital.education.gov.uk])
```

Comment:
added digital.education.gov.uk so we can register and sign in for accounts using our digital.education.gov.uk emails